functions/go/apply-replacements/README.md

```diff
@@ -0,0 +1,107 @@
+# apply-replacements
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Invoke the [kustomize replacements] feature as a KRM function. 
```

Comment:
`Invoke` implies it invokes kustomize's binary directly. And it's not the case here.

Reply:
changed `invoke` to `use`, thank you for the suggestion!